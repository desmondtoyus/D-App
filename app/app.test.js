import React from 'react'
import {renderWithProviders, cleanup} from '../__tests__/setup/test-setup'
import App from './App'

 afterEach(cleanup)


 it('should take a snapshot', () => {
    const { asFragment } = renderWithProviders(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
   });

// test("Wrapper not logged in", () => {
//   const { getByText } = renderWithProviders(<Wrapper />, {
//     store: { user: null }
//   });
//   getByText("login");
//   getByText("register");
// });

// test("Wrapper logged in", () => {
//   const { getByText } = renderWithProviders(<Wrapper />, {
//     reduxState: {
//       user: {
//         name: "bob"
//       }
//     }
//   });

//   getByText("bob");
// });