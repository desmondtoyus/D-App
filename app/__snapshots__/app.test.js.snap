// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      HEADER - Desmond Ademiluyi - 0
    </div>
    <div>
      <h1>
        Welcome Home Son 2 
      </h1>
      <button>
        Increase counter 1
      </button>
    </div>
    <div>
      FOOTER @2020 Desmond Ademiluyi 
    </div>
  </div>
</DocumentFragment>
`;
